config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
+  get "/", to: "welcome#index"
+  get "/login", to: "login#index"
```

Comment:
So the route will look like `"/login, to #sessions#new"`

Reply:
Agree as well, I will be using these sessions for the login/out user stories.